### ISSUE TITLE
@emits links to method if the event has the same name

### ISSUE THREAD

**Issue by VividVisions** [EnterTheNameHere/esdoc-monorepo#30]
Is there a way to differentiate an event from a method if they have the same name (other than renaming one or both)?
How to document the event itself?

**Comment by EnterTheNameHere** [comment]
If you have a simple example of code of such situation, it would be appreciated.

**Comment by EnterTheNameHere** [comment]
Yes, this should be easily detected by ESDoc as two different kinds, The `@variation` tag should also work, but it's not implemented. This can be hacked in, if needed, otherwise updated JSDoc parsing and detection will fix it in future.
